_data/_events.json

```diff
@@ -0,0 +1,239 @@
+[
+   {
+      "name":"",
+      "id":"",
+      "title":"",
+      "description":"",
+      "callsForPapersDeadline":"",
+      "location":"",
+      "startDate":"",
+      "endDate":"",
+      "date":"",
+      "openingParagraph":"",
+      "pastOpeningParagraph":"",
+      "blockquotes":[
+         ""
+      ],
+      "urlsAfterBlockquote":[
+         ""
+      ],
+      "locationHeader":"",
+      "locationName":"",
+      "linkHeader":"",
+      "linkUrl":[
+         ""
+      ],
+      "impDatesHeader":"",
```

Comment:
What is this?

If `imp` stands for "important", let's just write `importantDatesHeader`.

Reply:
I think we just want:
```
"importantDates": [ {
        "name": "...",
        "date": "..."
    },
    ...
]
```